docs/mathematics/tetrahedral-number.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Tetrahedral Number
+slug: /mathematics/tetrahedral-number
+---
+
+## Introduction
+
+A Tetrahedral number is a figurate number. A Figurate number is a number that can be represented by a regular geometrical arrangement of equally spaced points. The Tetrahedral number represents a pyramid with a triangular base and three sides. We are given a number `n` and we need to print the `nth` tetrahedral number.
+
+## Explanation 
+
+The `nth` Tetrahedral number is the sum of first `n` triangular numbers. For calculating the `nth` tetrahedral number just apply the formula given below. First few tetrahedral numbers are `1`, `4`, `10`, `20`, `35`, `56`, `84`, `120`, `165`, `220`.
+
+<img src="https://i.imgur.com/pOM3qXy.jpg" width="272" height="189" /><br/><br/>
```

Comment:
Make sure that the image used is your original creation. And can you please change the colour of the numbers as it may get difficult for users to read it in the dark mode.

Reply:
Sir image is added by me

Reply:
Numbers color changed in image